app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -28,6 +31,16 @@ class AdvancedFilter extends React.Component {
   }
 
   componentDidMount() {
+    getAllLanguageDisplayNames(this.props.authenticity_token, displayNames => {
```

Comment:
the advanced filter for primary language seems to be returning no results now. haven't really looked into it but I'm guessing it could be because the display name needs to be converted back to the language code for the query? this also has implications on existing advanced filters with primary language set because we'll need to modify their saved filters/user export presets to change their primary language filter values from `English` to `:eng` if that's what the value of it is in the db

Reply:
Ah, good catch! We'll need to make sure you can still search on the full terms.